packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,57 @@
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #f4f4f4);
+    border-bottom: var(--terra-action-header-border-bottom, 1px solid #dedfe0);
+    border-left: var(--terra-action-header-border-left);
+    border-right: var(--terra-action-header-border-right);
+    border-top: var(--terra-action-header-border-top);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
```

Comment:
Found that the action-header height is not consistent without buttons or menu (also when completely empty).  

- add themeable miniumum height on **.flex-header**: 
`min-height: var(--terra-action-header-min-height, 3.6428571429rem);`

- add themeable padding-top or margin-top to **.title**: 
`margin-top: var(--terra-action-header-title-margin-top, 0.1428571429rem);`